### PR TITLE
Update the aws integration for route53

### PR DESCRIPTION
**Pull request by deterralba**
Trello [card](https://trello.com/c/YoqaHGKQ/9-enhance-route53-to-include-tags-and-latency-metrics)

Update the doc for the route53 crawler. Add the IAM policy.

### CODE REVIEW

File: content/integrations/aws.md [hunk 1]
```diff
@@ -56,6 +56,7 @@ There are a number of other AWS services that are also available in Datadog but
 | Lambda |
 | MachineLearning |
 | OpsWorks |
+| Route53 |
```
Comment (tmichelet) [inline content/integrations/aws.md]: What is this for?

Comment (deterralba) [inline content/integrations/aws.md]: @tmichelet To appear in that list here http://docs.datadoghq.com/integrations/aws/ : 

screen shot 2016-05-12 at 3 13 01 pm

**Comment by tmichelet** [comment]
👍 could you merge that into staging and release it on staging? (there is a jenkins job for that)

**Comment by deterralba** [comment]
@tmichelet [Done](http://docs-staging.datadoghq.com/integrations/aws/).
It seems that merging master into staging hadn't been done any time recently.

**Comment by tmichelet** [comment]
👍

**Comment by deterralba** [comment]
@tmichelet Matt told me that staging was barely ever used.

**Comment by deterralba** [comment]
Will merge tomorrow

**Pull request merged**